### ISSUE TITLE
After entering the trained command line, the program is stuck.

### ISSUE THREAD

**Issue by ly1035327995** [michuanhaohao/reid-strong-baseline#223]
Loading pretrained ImageNet model......
Train without center loss, the loss type is triplet
label smooth on, numclasses: 751
2023-10-25 15:36:07,901 reid_baseline.train INFO: Start training
D:\ProgramData\Anaconda3\envs\ReID\lib\site-packages\ignite\handlers\checkpoint.py:971: UserWarning: Argument save_interval is deprecated and should be No
ne. This argument will be removed in 0.5.0.Please, use events filtering instead, e.g. Events.ITERATION_STARTED(every=1000)
  warnings.warn(msg)
D:\ProgramData\Anaconda3\envs\ReID\lib\site-packages\torch\optim\lr_scheduler.py:134: UserWarning: Detected call of `lr_scheduler.step()` before `optimize
r.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this wi
ll result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-
learning-rate
The program gets stuck. Nothing comes out.
